cmd/contrib/main.go

```diff
@@ -137,6 +137,17 @@ func newPlugin() error {
 
 				return nil
 			},
+			Transform: func(ans interface{}) (newAns interface{}) {
```

Comment:
This change is perfect! Love that we're automatically changing this for the user (rather than making it part of the validation and return an error message), that way we're not burdening the user with doing more work than needed. :raised_hands: